src/Library/Plugin/Plugin.php

```diff
@@ -333,6 +373,64 @@ public function getSlug() {
 		return $this->slug;
 	}
 
+	/**
+	 * Create page objects based on $this->config parameter
+	 *
+	 * @since SINCEVERSION
+	 * @access private
+	 */
+	private function setPages() {
+		$pages = [];
+		if ( isset( $this->pluginConfig['pages'] ) ) {
+			foreach ($this->pluginConfig['pages'] as $page ) {
```

Comment:
Watch for spacing.

Instead of this:
`foreach ($this->pluginConfig['pages'] as $page ) {`
... do this:
`foreach ( $this->pluginConfig['pages'] as $page ) {`

Reply:
Corrected. Those are so easy to miss.